src/Blef.GameLogic/PokerHands/HighStraight.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Blef.GameLogic.PokerHands
+{
+    /// <summary>
+    /// Duży strit
+    /// </summary>
+    public class HighStraight : PokerHand
+    {
+        private readonly Card card;
```

Comment:
Po co w stricie karta? Strit to strit. Nie wymaga podawania ani koloru ani rangi karty. Jedyne co to jest strit maly i duzy analogicznie do pokera malego i duzego.